### PR TITLE
Add 'forceStop' NB CLI option in NB5 

### PR DESCRIPTION
**Pull request by yabinmeng**
... port over from similar functionality in NB4 (PR# [675](https://github.com/nosqlbench/nosqlbench/pull/675))

### CODE REVIEW

File: engine-core/src/test/java/io/nosqlbench/engine/core/ActivityExecutorTest.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public synchronized void testRestart() {
         try {
             ad.setThreads(1);
             ae.startActivity();
-            ae.stopActivity();
+            ae.stopActivity(false);
```
Comment (jshook) [inline engine-core/src/test/java/io/nosqlbench/engine/core/ActivityExecutorTest.java:68]: I'm not a huge fan of using booleans to encode critical details like this. It's not very self explanatory.
Can we break this out into an `ae.forceStopActivity()` for example?

**Review by jshook — CHANGES_REQUESTED** [review]
looks good in general except for the boolean flag. It would be better to have a different method here.

File: engine-core/src/test/java/io/nosqlbench/engine/core/ActivityExecutorTest.java (hunk 1 above)
Comment (yabinmeng) [reply engine-core/src/test/java/io/nosqlbench/engine/core/ActivityExecutorTest.java:68]: I will make the changes.

Comment (yabinmeng) [reply engine-core/src/test/java/io/nosqlbench/engine/core/ActivityExecutorTest.java]: @jshook change have been made. A separate forceStopActivity() method is added.

**Review by jshook — APPROVED** [review]
LGTM!

**Comment by jshook** [comment]
Thanks, @yabinmeng !

**Pull request merged**